src/rendering/framebufferrenderer.cpp

```diff
@@ -66,955 +66,963 @@ namespace {
+            GL_FRAMEBUFFER,
+            GL_COLOR_ATTACHMENT0,
+            GL_TEXTURE_2D,
+            nOneStripTexture,
+            0
+        );
 
-    glGenVertexArrays(1, &nOneStripVAO);
-    glBindVertexArray(nOneStripVAO);
-    glGenBuffers(1, &nOneStripVBO);
-    glBindBuffer(GL_ARRAY_BUFFER, nOneStripVBO);
-    glBufferData(
-        GL_ARRAY_BUFFER,
-        sizeof(GLfloat) * _nAaSamples * (NVERTEX + 12),
-        nOneStripVertexData,
-        GL_STATIC_DRAW
-    );
-
-    // position
-    glVertexAttribPointer(
-        0,
-        4,
-        GL_FLOAT,
-        GL_FALSE,
-        sizeof(GLfloat) * 6,
-        nullptr
-    );
-    glEnableVertexAttribArray(0);
-
-    // texture coords
-    glVertexAttribPointer(
-        1,
-        2,
-        GL_FLOAT,
-        GL_FALSE,
-        sizeof(GLfloat) * 6,
-        reinterpret_cast<GLvoid*>(sizeof(GLfloat) * 4)
-    );
-    glEnableVertexAttribArray(1);
-    delete[] nOneStripVertexData;
-
-    // fbo texture buffer
-    glGenTextures(1, &nOneStripTexture);
-    glBindTexture(GL_TEXTURE_2D, nOneStripTexture);
-    glTexImage2D(
-        GL_TEXTURE_2D,
-        0,
-        GL_RGBA32F,
-        _nAaSamples,
-        ONEPIXEL,
-        0,
-        GL_RGBA,
-        GL_FLOAT,
-        nullptr
-    );
-
-    glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
-    glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
-
-    glGenFramebuffers(1, &nOneStripFramebuffer);
-    glBindFramebuffer(GL_FRAMEBUFFER, nOneStripFramebuffer);
-    glFramebufferTexture2D(
-        GL_FRAMEBUFFER,
-        GL_COLOR_ATTACHMENT0,
-        GL_TEXTURE_2D,
-        nOneStripTexture,
-        0
-    );
-
-    status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
-    if (status != GL_FRAMEBUFFER_COMPLETE) {
-        LERROR("nOneStrip framebuffer is not complete");
-    }
+        status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
+        if (status != GL_FRAMEBUFFER_COMPLETE) {
+            LERROR("nOneStrip framebuffer is not complete");
+        }
 
-    glViewport(0, 0, _nAaSamples, ONEPIXEL);
+        glViewport(0, 0, _nAaSamples, ONEPIXEL);
 
-    std::unique_ptr<ghoul::opengl::ProgramObject> nOneStripProgram = nullptr;
-    try {
-        nOneStripProgram = ghoul::opengl::ProgramObject::Build(
-            "OneStrip MSAA",
-            absPath("${SHADERS}/framebuffer/nOneStripMSAA.vert"),
-            absPath("${SHADERS}/framebuffer/nOneStripMSAA.frag")
-        );
-    }
-    catch (const ghoul::RuntimeError& e) {
-        LERRORC(e.component, e.message);
-    }
+        std::unique_ptr<ghoul::opengl::ProgramObject> nOneStripProgram = nullptr;
+        try {
+            nOneStripProgram = ghoul::opengl::ProgramObject::Build(
+                "OneStrip MSAA",
+                absPath("${SHADERS}/framebuffer/nOneStripMSAA.vert"),
+                absPath("${SHADERS}/framebuffer/nOneStripMSAA.frag")
+            );
+        }
+        catch (const ghoul::RuntimeError& e) {
```

Comment:
No need to catch the exception;  if it throws, the next access will be to a `nullptr` anyway

Reply:
Removed.